modules/phase_field/include/postprocessors/FindValueOnLine.h

```diff
@@ -0,0 +1,64 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef FINDVALUEONLINE_H
+#define FINDVALUEONLINE_H
+
+#include "GeneralPostprocessor.h"
+#include "Coupleable.h"
+
+class FindValueOnLine;
+
+template<>
+InputParameters validParams<FindValueOnLine>();
+
+/**
+ * Finds a secific value along a line through a bisection search/
```

Comment:
I think this doco needs expanding.  I didn't know what it meant till i read the code.  Also i think you need to state somewhere that you're assuming monotonicity of coupled_var as a user (me) will just use this post processor without thinking about such things.